browser-test/src/support/admin_programs.ts

```diff
@@ -483,6 +483,16 @@ export class AdminPrograms {
       .isVisible()
   }
 
+  async getStatusOption(): Promise<string> {
+    const selectedValue = this.applicationFrame()
+      .locator('.cf-program-admin-status-selector')
```

Comment:
Not blocking: I think we might be able to restructure the DOM a bit so that this selector isn't necessary if you'd like. If we put the "Statuses:" text inside an `<label>` and note that it's a field for the select option, then Play will allow you to do a selector for a label with text "Statuses". InputValue will automatically resolve this to the associated input (https://playwright.dev/docs/api/class-locator#locator-input-value).

Benefits:
  * Fewer reference classes
  * Semantically more correct for screen readers and A11Y.

Concretely, the HTML would be something like:
```
<div>
  <label class="cf-program-admin-status-selector-label" for="status-options">Statuses:</label>
  <select name="status-options">
   <option>...</option>
  </select>
</div>
```

Asserting whether the label appears:
```
await this.applicationFrame()
      .locator('.cf-program-admin-status-selector-label').isVisible()
```

Getting the value:
```
await this.applicationFrame()
      .locator('.cf-program-admin-status-selector-label').inputValue()
```

Reply:
ah thanks!  I switched to label with for.  When I couldn't use our SelectWithLabel (due to the formatting) I accidentally through label out with the bathwater completely.